drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTablePresenter.java

```diff
@@ -317,24 +329,42 @@ public void refreshContent( final ObservablePath path,
         this.oracle = oracleFactory.makeAsyncPackageDataModelOracle( path,
                                                                      model,
                                                                      dataModel );
-        this.lockManager = getLockManager( path,
-                                           placeRequest );
         this.access.setReadOnly( isReadOnly );
         this.rm = new BRLRuleModel( model );
 
-        //Ensure field data-type is set (field did not exist before 5.2)
-        for ( CompositeColumn<?> column : model.getConditions() ) {
-            if ( column instanceof Pattern52 ) {
-                final Pattern52 pattern = (Pattern52) column;
-                for ( ConditionCol52 condition : pattern.getChildColumns() ) {
-                    condition.setFieldType( oracle.getFieldType( pattern.getFactType(),
-                                                                 condition.getFactField() ) );
-                }
-            }
-        }
+        this.uiModel = makeUiModel();
+        this.view = makeView( workItemDefinitions );
+
+        initialiseLockManager();
+        initialiseUtilities();
+        initialiseModels();
+        initialiseValidationAndVerification();
+        initialiseAuditLog();
+    }
+
+    //Setup LockManager
+    void initialiseLockManager() {
+        lockManager.init( new LockTarget( currentPath,
+                                          parent.getView().asWidget(),
+                                          placeRequest,
+                                          new LockTarget.TitleProvider() {
+                                              @Override
+                                              public String getTitle() {
+                                                  return currentPath.getFileName() + " - " + resourceType.getDescription();
+                                              }
+                                          },
+                                          new Runnable() {
+                                              @Override
+                                              public void run() {
+                                                  //Nothing to do
+                                              }
+                                          } ),
```

Comment:
last two parameters to LockTarget constructor could be lambda expressions to make this statement shorter.